### PR TITLE
change arrow color, dynamic image size

### PR DESCRIPTION
**Pull request by dchen150**
## Description
#55, #72. Arrow changed to primary color. Width is dynamic depending on viewport width and height.
image

### CODE REVIEW

**Comment by anneguo3** [comment]
@dchen150 when I went to the staging PR it still only lets me navigate through the photos based on clicking the arrow exactly rather than just on the photo area

**Comment by kevin-zou** [comment]
Does this show up for you locally? The diamonds look like they've seen better days :'(
image

If not it's probably just the deploy being weird again

**Comment by dchen150** [comment]
@anneguo3 will work on it. @kevin-zou ya  idk why it does that. When you expand it it goes back to regular diamonds. Do you think importing an SVG instead of making the diamonds out of css would be better? Or is it worth investing time in figuring out why the diamonds are doing that when the window shrinks

**Comment by kevin-zou** [comment]
Oh I think I found out why. The `width: 30%` property on its container makes it distorted. Removing this seems to fix it - can you try this and see if it doesn't break anything?

File: components/Carousel.js [hunk 1]
```diff
@@ -34,33 +33,37 @@ const EmptyDiamond = styled.div`
...
-    width: ${(props) => props.width};
-    height: ${(props) => props.height};
+    width: 29vw;
+    height: 25hw;
     margin-bottom: 10px;
     border-radius: 6px;
     object-fit: cover;
+    ${(p) => p.theme.mediaQueries.mobile} {
+        width: 100%;
+        height: 100%;
```
Comment (kevin-zou) [inline components/Carousel.js:47]: Can we set a static height like we did for the desktop view? On mobile the different heights between images leads to a "jumpy" experience whenever the picture changes!

**Review by kevin-zou — APPROVED** [review]
Anne's described behaviour on-click of the entire image changing the image isn't implemented yet, but everything else looks good to me 🔥

**Review by panjenny0 — APPROVED** [review]
lgtm!

**Comment by anneguo3** [comment]
@dchen150 i'm on a widescreen monitor and...
image

**Pull request merged**